recipes/libssh2/bld.bat

```diff
@@ -0,0 +1,9 @@
+call setup_x64.bat
```

Comment:
I switched to NMake Makefiles already. Was fun to make it work with patch, though.